phpBB/styles/prosilver/theme/icons.css

```
@@ -27,6 +27,8 @@ blockquote cite:before,
 	text-rendering: auto; /* optimizelegibility throws things off #1094 */
 }
 
+.icon { padding-right: 2px; }
```

Oh, if you could change it to not be on one line, then I can merge it.
```suggestion
.icon { 
	padding-right: 2px;
}
```